### PR TITLE
Add admin controls for removing archives and sending email to users

### PR DESCRIPTION
**Pull request by taravancil**

### CODE REVIEW

File: assets/html/archive.html [hunk 1]
```diff
@@ -79,7 +79,7 @@
           </li>
         </ul>
       </div>
-      <% if (isOwner) { %>
+      <% if (isOwner || sessionUser.scopes.includes('admin:dats')) { %>
```
Comment (pfrazee) [inline assets/html/archive.html]: This doesn't work, unfortunately. The /v1/archives/remove route operates on the session user.

We should use a separate link entirely, that indicates it's an admin action, and we should add a route to the admin API in hypercloud which removes the archive.

**Review by pfrazee — CHANGES_REQUESTED** [review]

**Comment by pfrazee** [comment]
Need to update the remove-archive bit (or maybe just remove it and do in a separate PR). Email form looks good though!

**Comment by taravancil** [comment]
Will open a separate PR for the remove archive stuff

**Comment by taravancil** [comment]
Making this PR directly to beakerbrowser/hashbase. Closing.

**Pull request closed without merging**